### PR TITLE
Mii Brawler followup adjustments

### PR DESCRIPTION
**Pull request by SkewedAskew**
### Earthquake Fist:
- (!) Can now be B-reversed
- Ground
  - (*) Fixed charging ending prematurely if close to an edge
  - [+] Bottom hitbox lengthened by 3.0u
  - [+] Added hand hitbox
  - [/] Max Charge Frames: 15 -> 20
  - [+] Max Distance: 25 -> 35
  - [$] Added visual indicator of where the quake will appear
  - [$] Adjusted vfx and sfx
  - [+] Hitbox Duration: F13-14 -> F13-15
  - Quake
    - [-] Damage: 18.0/16.0% -> 12.0%
    - [R] BKB: 80/82 -> 75
    - [R] KBG: 48/53 -> 28
    - [/] Hitlag Multiplier: 1.1/1.0x -> 1.0x
  - Hand
    - Damage: 8.0%
    - Angle: 45
    - BKB: 60
    - KBG: 60
    - Hitbox Size: 3.0u
- Air
  - [$] Adjusted vfx 

### Wild Throw:
- (!) Can now be B-reversed

https://github.com/HDR-Development/HewDraw-Remix/assets/55216313/81cb74ff-cff0-4ee8-919b-b8bebd062998

### CODE REVIEW

**Comment by MosaMosa-x** [comment]
so cool

File: fighters/miifighter/src/acmd/specials.rs [hunk 1]
```diff
@@ -496,14 +496,11 @@ unsafe fn miifighter_special_lw1_game(fighter: &mut L2CAgentBase) {
...
-        ATTACK(fighter, 1, 0, Hash40::new("top"), 16.0, 90, 53, 0, 82, 5.0, 0.0, 4.0, 12.0 + charge_distance, Some(0.0), Some(13.0), Some(12.0 + charge_distance), 1.0, 0.0, *ATTACK_SETOFF_KIND_OFF, *ATTACK_LR_CHECK_POS, true, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_L, *COLLISION_SOUND_ATTR_KICK, *ATTACK_REGION_KICK);
+        ATTACK(fighter, 0, 0, Hash40::new("top"), 12.0, 90, 28, 0, 75, 5.0, 0.0, 4.5, 8.0 + charge_distance, Some(0.0), Some(4.5), Some(16.0 + charge_distance), 1.0, 0.0, *ATTACK_SETOFF_KIND_OFF, *ATTACK_LR_CHECK_POS, true, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_L, *COLLISION_SOUND_ATTR_HEAVY, *ATTACK_REGION_KICK);
```
Comment (jobrien97) [inline fighters/miifighter/src/acmd/specials.rs]: Is the 0.0 SDI multiplier intentional/necessary?

File: fighters/miifighter/src/opff.rs [hunk 2]
```diff
@@ -65,8 +98,8 @@ unsafe fn earthquake_punch(fighter: &mut L2CFighterCommon, boma: &mut BattleObje
         let is_hold = ControlModule::check_button_on(fighter.module_accessor, *CONTROL_PAD_BUTTON_SPECIAL);
         let charge = VarModule::get_int(fighter.battle_object, vars::miifighter::status::SPECIAL_LW1_CHARGE);
         let charge_distance = VarModule::get_float(fighter.battle_object, vars::miifighter::status::SPECIAL_LW1_CHARGE_DISTANCE) as f32;
-        let max_charge_frames = 15.0;
-        let max_charge_distance = 25.0;
+        let max_charge_frames = 20.0;
```
Comment (jobrien97) [inline fighters/miifighter/src/opff.rs]: Can we turn these into ParamModule params for maintainability?

Comment (SkewedAskew) [reply fighters/miifighter/src/opff.rs]: Done, I think

**Review by jobrien97 — APPROVED** [review]

**Pull request merged**